### PR TITLE
Uppy added for file upload

### PR DESCRIPTION
**Pull request by wvandolah**
# Description

Created component on frontend to allow for files to be uploaded. added middleware on backend to take files and send them to aws s3.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: frontend/src/components/__tests__/FileUpload.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    const { container } = render(<FileUpload />);
+    const buttons = container.querySelector('div');
+
+    expect(buttons).toBeInTheDocument();
+  });
+
+  test('should render passed text', () => {
+    const { container } = render(<FileUpload text='this is a thing' />);
+    const buttons = container.querySelector('button');
+    expect(buttons.textContent).toBe('this is a thing');
```
Comment (NandoTheessen) [inline frontend/src/components/__tests__/FileUpload.spec.tsx]: I believe we should at least be able to confirm that the modal is opening.
If you're unsure how to do that, and how the DOM changes when clicking the button you can use `debug()` that comes from the render function

File: backend/src/app.ts [hunk 2]
```diff
@@ -69,6 +71,23 @@ server.route('/itemComplete').post(items.itemComplete);
 
 server.route('/email').post(email.send);
 
+const options = {
```
Comment (NandoTheessen) [inline backend/src/app.ts]: Is there anything about testing in the uppy documentation? 
Feels a bit like a black box looking at it.

Comment (wvandolah) [reply backend/src/app.ts]: I can't find anything.  I also can't find how to turn off all those console.log on the front-end.

File: frontend/src/components/__tests__/FileUpload.spec.tsx (hunk 1 above)
Comment (wvandolah) [reply frontend/src/components/__tests__/FileUpload.spec.tsx]: Working on it now

**Pull request merged**